src/python/pants/option/config.py

```diff
@@ -14,10 +14,15 @@
 from twitter.common.collections import OrderedSet
 
 from pants.base.build_environment import get_buildroot, get_pants_cachedir, get_pants_configdir
+from pants.base.deprecated import deprecated_conditional
+from pants.option.global_options import GlobalOptionsRegistrar
 from pants.util.eval import parse_expression
 from pants.util.meta import AbstractClass
 
 
+GLOBAL_SECTION = 'GLOBAL'
```

Comment:
Should this just use `from pants.option.arg_splitter import GLOBAL_SCOPE_CONFIG_SECTION`? It looks like that introduces no circular deps.

Reply:
sure, done.